### ISSUE TITLE
Questions: How do I add the original tags? What is the process for selecting a representative slice?

### ISSUE THREAD

**Issue by deepbased** [PantelisGeorgiadis/dcmjs-imaging#7]
Hi @PantelisGeorgiadis ,
Wishing you a happy new year:) 
There are two questions I would like to ask (not issues):

1. Is it possible to add or rebuild the original DICOM tags after the image has been decoded? Maybe as an EXIF tag? In the process, all Patient/Series/Study tags disappear, or maybe I'm missing something.

2. What is the process of selecting a representative slice from 30 or more slices? Is the slice with the highest mean intensity chosen as the representative slice by taking the mean value of all pixel intensities in each slice?

Thanks!

**Comment by PantelisGeorgiadis** [comment]
Hello @deepbased,
Happy New Year!

Regarding your questions,
1. Yes! The `getElements` and `setElement` functions of the `DicomImage` class, will help you get all original DICOM tags and add/update them, respectively. I just verified that after decoding/rendering an image, all original DIICOM tags are still there and can be accessed by using `getElements`.
2. I believe that it is not in the best of your interest to go through all pixels for each slice, just to select a representative slice, except if this is really crucial for you. As you understand this would require time and resources. Most of the application I've seen are picking a representative slice base on a fraction of the number of slices (e.g. 1/3).

Hope I helped!
